src/Models/Services/TelemetryService/TelemetryService.ts

```diff
@@ -1,17 +1,26 @@
-import { Telemetry } from './Telemetry';
+import { getDebugLogger } from '../Utils';
+import { ITelemetryEvent } from './Telemetry';
+
+const debugLogging = false;
+const logDebugConsole = getDebugLogger('TelemetryService', debugLogging);
 
 class TelemetryService {
-    static telemetryCallback;
+    static telemetryCallback: (telemetry: ITelemetryEvent) => void;
 
     // Attach telemetry processing callback from consuming application
     static registerTelemetryCallback(
-        telemetryCallback: (telemetry: Telemetry) => void
+        telemetryCallback: (telemetry: ITelemetryEvent) => void
```

Comment:
Since there is a specific type 'event' telemetry down in `TelemetryType`, it is confusing to see ITelemetryEvent here, can we change it to `ITelemetry` or something else?